Website/app/contact/page.js

```diff
@@ -10,38 +10,44 @@ export default function contact() {
-          We&apos;d love to hear from you!
-        </p>
+      <div className="select-none bg-gradient-to-b from-top via-about to-bottom ">
+      
+        <div className="flex justify-center items-center  w-full max-md:p-2   p-12">
+
         <form
-          className="grid grid-cols-2 justify-items-center"
+          className="md:w-3/5   bg-white rounded-2xl shadow-lg shadow-gray-500 p-14" 
           method="POST"
           action="https://formsubmit.co/e94b201f9a607081e9f9f8ee09ff5e25"
+          aria-label="Contact Us Form"
         >
+                      <h1 className="  max-md:text-3xl mb-4 text-5xl font-bold">Contact Us</h1>
+                      <p className="">          We&apos;d love to hear from you!
+                      </p>
+                      <div className="flex justify-around gap-2 mt-6 max-md:flex-col">
           <input
-            className="bg-white rounded-md focus:outline-blue-600 text-blue-600 outline:none caret-blue-600 md:w-6/12 sm:w-1/2 h-14 p-3 col-span-2"
+            className= "border max-md:w-full rounded-full max-w-[400px] p-2 w-1/2 bg-white  focus:outline-blue-600 text-blue-600 outline:none caret-blue-600 md:w-6/12 sm:w-1/2  h-12  col-span-2"
             type="text"
             name="name"
             autoFocus={true}
             inputMode="text"
             required
             placeholder="Name"
+            pattern="^[A-Za-z\s]{1,}$"
           />
-          <input
-            className="bg-white rounded-md focus:outline-blue-600 outline:none caret-blue-600 mt-10 md:ml-auto md:w-1/2 justify-end h-14 md:col-span-1 col-span-2 p-2 text-blue-600"
+          <input 
+            className="border max-md:w-full max-md:mt-6  max-w-[400px] rounded-full p-2 w-1/2 bg-white  focus:outline-blue-600 outline:none caret-blue-600  h-12  md:ml-auto md:w-1/2 justify-end  md:col-span-1 col-span-2  text-blue-600"
             inputMode="email"
-            pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/"
+            pattern="^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$"
             type="email"
             name="email"
             required
             placeholder="Email"
           />
+          </div>
           <input
-            className="bg-white rounded-md focus:outline-blue-600 outline:none caret-blue-600 mt-10 md:mr-auto md:ml-1 md:w-1/2 justify-start  h-14 p-2  md:col-span-1 col-span-2 text-blue-600"
+            className="border rounded-full p-2 w-full my-2 bg-white  focus:outline-blue-600 outline:none caret-blue-600 mt-6 md:mr-auto md:ml-1 justify-start  h-12   md:col-span-1 col-span-2 text-blue-600"
             minLength={6}
-            pattern="/^([+]?\d{1,2}[-\s]?|)\d{3}[-\s]?\d{3}[-\s]?\d{4}$/"
+            pattern="^[+]?[(]?[0-9]{3}[)]?[-\s.]?[0-9]{3}[-\s.]?[0-9]{4,6}$"
```

Comment:
Do you think this change would be good? Try the possibilities and tell me.
```suggestion
            pattern="^(\+\d{1,3}[- ]?)?\d{10}$"
```

Reply:
yeah ^(\+\d{1,3}[- ]?)?\d{10}$" this would be better according to me